test/robosherlock_utest/ImageSegmentationAnnotator_utest.cpp

```diff
@@ -0,0 +1,42 @@
+#include <gtest/gtest.h>
+#include <rs/scene_cas.h>
+#include <rs/flowcontrol/RSAnalysisEngine.h>
+#undef OUT_LEVEL
+#define OUT_LEVEL OUT_LEVEL_DEBUG
+#include "../main.h"
+
+
+
+void imageSegmentationAnnotatorTest()
+{
+
+  std::vector<std::string> engineList = {"CollectionReader","ImagePreprocessor","NormalEstimator","PlaneAnnotator","PointCloudClusterExtractor","ImageSegmentationAnnotator"};
```

Comment:
no need for PointCloudClusterExtractor here; it shadows the results of ImageSegmentationaAnnotator, and the check clusters.size()>0 will not reflect the ImageSegm.Annotator